lib/linguist/languages.yml

```diff
@@ -303,6 +305,7 @@ Bison:
   extensions:
   - .bison
   ace_mode: text
+  color: "#6A463F"
```

Comment:
I don't object to Bison getting a color.  However, it seems unlikely that Bison will diverge significantly from Yacc.  This is from the Bison manual:

> Bison is upward compatible with Yacc: all properly-written Yacc grammars ought to work with Bison with no change. Anyone familiar with Yacc should be able to use Bison with little trouble.